server/src/resolvers/Query.js

```diff
@@ -0,0 +1,9 @@
+const query = {
```

Comment:
Same suggestion as before
```suggestion
const Query = {
```